DIPs/DIP1009.md

```diff
@@ -42,38 +44,51 @@ body
     ulong ret = 0;
 }
 ```
-A second reason contracts are hard to use is that they require the use of the keyword `do` (or `body`, see [DIP1003](https://github.com/dlang/DIPs/blob/master/DIPs/DIP1003.md)) before the function body. Any way of writing them that could eliminate this need would be an improvement.
 
-Finally, `in` and `out` contracts in D require explicitly writing `assert` statements, or some equivalent functionality, in their code block in order to perform checks. But because of the nature of contracts, `assert` can be assumed in the vast majority of cases, and it is tedious to have to write it specifically. For the common case, the mere presence of `in` or `out` is sufficient. For those programmers who require the use of other forms of checking, the existing syntax will remain as is. 
+2. Contracts require the use of the keyword `do` (or `body`, see [DIP1003](https://github.com/dlang/DIPs/blob/master/DIPs/DIP1003.md)) before the function body. Any way of writing them that eliminated this need would be an improvement.
+
+3. `in` and `out` contracts in D require explicitly writing `assert` statements, or some equivalent, in their code block in order to perform checks. But because of the nature of contracts, `assert` can be assumed in the vast majority of cases, and it is tedious to have to write it specifically. For the common case, the mere presence of `in` or `out` is sufficient. For those programmers who require the use of other forms of checking, the existing syntax will remain unchanged. 
 
-History shows that a small change in syntax can dramatically change the usability of a language feature. By adding a new syntax for contracts, an increase in their usage can be expected.
+Experience demonstrates, as in the case of `=>` lambdas, that a small change in syntax can dramatically affect the usage of a language feature. By adding a new syntax for contracts, a similar increase in their usage can be expected.
```

Comment:
"Experience demonstrates" isn't much different than "History shows". You do provide an example, but there's nothing to back it up. What's your source for the claim that the addition of the `=>` syntax increased language usage? If you can cite something that supports that claim, add it. If not, then it's probably best to cut this line entirely.